src/components/events/createEvent/EventsGeneralInfo.jsx

```diff
@@ -128,47 +182,92 @@ const EventsGeneralInfo = () => {
               <br />
               <Col span={8}>
-                <Checkbox value="First Aid Training">First Aid Training</Checkbox>
+                <Checkbox value="First Aid Training" onChange={() => setFirstAidTraining(true)}>
+                  First Aid Training
+                </Checkbox>
               </Col>
               <Col span={8}>
-                <Checkbox value="Serve Safe Knowledge">Serve Safe Knowledge</Checkbox>
+                <Checkbox value="Serve Safe Knowledge" onChange={() => setServeSafeKnowledge(true)}>
+                  Serve Safe Knowledge
+                </Checkbox>
               </Col>
               <Col span={8}>
-                <Checkbox value="Transportation Experience">Transportation Experience</Checkbox>
+                <Checkbox
+                  value="Transportation Experience"
+                  onChange={() => setTransportationExperience(true)}
+                >
+                  Transportation Experience
+                </Checkbox>
               </Col>
               <br />
               <br />
               <Col span={8}>
-                <Checkbox value="Moving / Warehouse Experience">
+                <Checkbox
+                  value="Moving / Warehouse Experience"
+                  onChange={() => setWarehouseExperience(true)}
+                >
                   Moving / Warehouse Experience
                 </Checkbox>
               </Col>
               <Col span={8}>
-                <Checkbox value="Food Service Industry Knowledge">
+                <Checkbox
+                  value="Food Service Industry Knowledge"
+                  onChange={() => setFoodServiceKnowledge(true)}
+                >
                   Food Service Industry Knowledge
                 </Checkbox>
               </Col>
             </Row>
           </Checkbox.Group>
         </Form.Item>
 
-        <Form.Item label="Location" rules={[{ required: true }]}>
-          <Input placeholder="Ex. Irvine, CA" />
+        <Form.Item label="Street Address" rules={[{ required: true }]}>
+          <Input
+            placeholder="Ex. 123 Banana St."
+            onChange={e => setEventAddressStreet(e.target.value)}
+          />
+        </Form.Item>
+        <Form.Item label="City" rules={[{ required: true }]}>
+          <Input placeholder="Austin" onChange={e => setEventAddressCity(e.target.value)} />
+        </Form.Item>
+        <Form.Item
+          label="State"
+          rules={[{ required: true }]}
+          onChange={e => setEventAddressState(e.target.value)}
+        >
+          <Input placeholder="TX" />
+        </Form.Item>
+        <Form.Item
+          label="Zipcode"
+          rules={[{ required: true }]}
+          onChange={e => setEventAddressZip(e.target.value)}
+        >
+          <Input placeholder="73301" />
         </Form.Item>
       </Form>
     </div>
   );
 };
 
+EventsGeneralInfo.propTypes = {
+  setStates: PropTypes.objectOf(PropTypes.any).isRequired,
```

Comment:
🚫 **[eslint]** <[react/forbid-prop-types](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/forbid-prop-types.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Prop type "any" is forbidden